Libraries/Components/Picker/PickerIOS.ios.js

```diff
@@ -159,7 +160,7 @@ const styles = StyleSheet.create({
     // The picker will conform to whatever width is given, but we do
     // have to set the component's height explicitly on the
     // surrounding view to ensure it gets rendered.
-    height: 216,
+    height: Platform.OS === 'ios' ? 216 : 27, // TODO(macOS ISS#2323203)
```

Comment:
Is 27 the size for NSComboBox with a controlSize of NSControlSizeRegular, NSControlSizeSmall, or NSControlSizeMini? We should at the very least document which size we're hardcoding

Reply:
In my debugging in a sample native app, I'm seeing the following heights for NSComboBox:

| Control Size | Intrinsic Height Constraint | AlignmentRectHeight     | Bounds Height     |
| :------------- | ----------: | ----------: | -----------: |
|  NSControlSizeRegular | 17 pts   | 17 pts    | 23 pts |
|  NSControlSizeSmall | 15pts   | 15pts    | 20 pts |
|  NSControlSizeMini | 14pts   | 14pts    | 20 pts |

Where's the 27 point height coming from?

Reply:
I just did a drag and drop of a Combo Box onto a Storyboard and looked at the inspector.   Although I just tried it again and now I see 25 🤷🏻‍♀️.  Maybe I misread it before.  I'll go with 25, it looks fine.   If the react height is larger the system renders the combo centered vertically.   But if its too small it starts to clip first the text then the chrome of the control itself.